firmware_DavidBoard/Core/Inc/TemperatureCalc.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+/**
+ * This module turns ADC voltage readings into temperature (in Celsius)
+ * Author(s): Jacob Smith
+ */
+
+#include "Common.h" //move these to TemperatureCalc.c
+#include "I2CManager.h"
+
+/***********************************************************************************************************************
+ * Definitions
+ **********************************************************************************************************************/
+
+#define ADC_QUANTIZATION 1024                     	//Number of unique ADC values
+#define ADC_V_FS 2048								//ADC full-scale voltage (mV)
+#define BOARD_V_FS 3.3                            	//Full-scale board voltage
+#define BOARD_V_FS_1000 3300						//Full-scale board voltage in mV
+#define PCB_DIVIDING_RESISTANCE 12000               //On-board resistor used for voltage division
+//Coeffs for Steinhart-Hart R-T equation of a thermistor
+//#define STEINHART_HART_A 0.00146162571738         //
+//#define STEINHART_HART_B 0.000239364498811        //
```

Comment:
There's a rule to where you should put defines. You gotta ask yourself, does the caller of my module care ? If the answer is yes, they go in the header file, if it's no, they go in the .c file. In your case, i seems all these parameters are irrelevant to the caller, so you should stick them in .c